bitcoincore/src/main/kotlin/io/horizontalsystems/bitcoincore/BitcoinCore.kt

```diff
@@ -163,7 +163,11 @@ class BitcoinCoreBuilder {
 
         val connectionManager = ConnectionManager(context)
 
-        val hdWallet = HDWallet(seed, network.coinType, purpose = bip.purpose)
+        var gapLimit = 20
```

Comment:
what's the gaplimit?

Reply:
It is used to determine how many addresses to query for when fetching transactions.